DEPS

```diff
@@ -139,7 +139,7 @@ allowed_hosts = [
 ]
 
 deps = {
-  'src': 'https://github.com/flutter/buildroot.git' + '@' + '0c64fd23eeedbad60c2bfa294710cd587ccb7c27',
+  'src': 'https://github.com/stuartmorgan/buildroot.git' + '@' + 'e3e15dea04be416be8bab2b551f41d8d10efc95b',
```

Comment:
Should this still be using your repo? Seems like the flutter repo is more up to date?

Reply:
I was testing https://github.com/flutter/buildroot/pull/362 to see if it fixed the compilation failures in the initial version, before sending that as a buildroot patch. Now that I know it does, I'll get that reviewed and landed and then update this.